inference-engine/src/low_precision_transformations/src/move_fake_quantize.cpp

```diff
@@ -0,0 +1,147 @@
+        operation_original_name = operation->get_friendly_name();
+        concat = operation->get_input_node_shared_ptr(0);
+        without_operation = false;
+    }
+    if (!ConcatTransformation::isQuantizedStatic(concat)) {
+        return false;
+    }
+    auto convert_q = (*fq->output(0).get_target_inputs().begin()).get_node()->shared_from_this();
+    std::shared_ptr<ov::Node> convert_dq, subtruct, multiply;
+    bool q_dq = false, with_subtract = false;
+    if (is_type<opset1::Convert>(convert_q)) {
+        q_dq = true;
+        convert_dq = (*convert_q->output(0).get_target_inputs().begin()).get_node()->shared_from_this();
+        subtruct = (*convert_dq->output(0).get_target_inputs().begin()).get_node()->shared_from_this();
+        if (!is_type<opset1::Subtract>(subtruct)) {
+            multiply = subtruct;
+        } else {
+            multiply = (*subtruct->output(0).get_target_inputs().begin()).get_node()->shared_from_this();
+            with_subtract = true;
+        }
+    }
+    std::vector<std::shared_ptr<ngraph::Node>> newNodes;
+    size_t input_size = concat->get_input_size();
+    for (size_t i{ 0 }; i < input_size; ++i) {
+        std::shared_ptr<ngraph::Node> fq_input;
+        if (without_operation) {
+            fq_input = concat->get_input_node_shared_ptr(i);
+        } else {
+            auto input = concat->get_input_node_shared_ptr(i);
+            if (is_type<opset1::Relu>(operation)) {
+                fq_input = std::make_shared<ngraph::opset1::Relu>(input);
+            }
+            fq_input->set_friendly_name(operation_original_name + std::to_string(i + 1));
+        }
+        auto newFq = fq->clone_with_new_inputs({ fq_input,
+            fq->get_input_node_shared_ptr(1),
+            fq->get_input_node_shared_ptr(2),
+            fq->get_input_node_shared_ptr(3),
+            fq->get_input_node_shared_ptr(4) });
+        newFq->set_friendly_name(fq_original_name + "_" + std::to_string(i + 1));
+        if (q_dq) {
+            auto newConvert_q = convert_q->clone_with_new_inputs({ newFq });
+            newConvert_q->set_friendly_name(convert_q_original_name + std::to_string(i + 1));
+            auto newConvert_dq = convert_dq->clone_with_new_inputs({ newConvert_q });
+            newConvert_dq->set_friendly_name(convert_dq_original_name + std::to_string(i + 1));
+            std::shared_ptr<ov::Node> newMultiply;
+            if (with_subtract) {
+                auto newSubtract = subtruct->clone_with_new_inputs({ newConvert_dq });
+                newSubtract->set_friendly_name(subtruct_original_name + std::to_string(i + 1));
+                newMultiply = multiply->clone_with_new_inputs({ subtruct, multiply->get_input_node_shared_ptr(1) });
+            } else {
+                newMultiply = multiply->clone_with_new_inputs({ newConvert_dq, multiply->get_input_node_shared_ptr(1) });
+            }
+            newMultiply->set_friendly_name(multiply_original_name + std::to_string(i + 1));
```

Comment:
please reuse `moveDequantizationAfter` or (if it's not possible) extract to new separate method `moveDequantizationBefore`